Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

*See also*:

[[restore]]
==== image:images/yes.png[yes] restore (verb)

> *Description*: To correct or replace data that has been lost or corrupted, after a problem in the cluster.

Reviewer: ```suggestion
*Description*: To _restore_ is to correct or replace data that was lost or corrupted after a problem occurred in a cluster.
```

*See also*:

[[restore]]
==== image:images/yes.png[yes] restore (verb)

> *Description*: To _restore_ is to correct or replace data that was lost or corrupted after a problem occurred in the cluster.

Reviewer 2: This definition of "restore" is cluster-specific. Update to add which product or situation this definition applies in.

Reviewer 3: Updated to "In Red Hat OpenShift"
